### PR TITLE
`[PRD]` Re-order nav links, remove proposal links

### PR DESCRIPTION
**Pull request by DarksightKellar**
Closes #2520

EDIT: Merge after merging in #2567 

Rearranging only. No copy updates, as per linked issue

### CODE REVIEW

File: src/components/ui/page/Navigation/NavigationTooltip.tsx
Comment (DarksightKellar) [inline src/components/ui/page/Navigation/NavigationTooltip.tsx:1]: Noticed this file has no refs

**Review by adamgall — COMMENTED** [review]
Can probably delete the entire Proposals page too then, no?

**Comment by DarksightKellar** [comment]
> Can probably delete the entire Proposals page too then, no?

@adamgall I was just about to do this, then saw it's literally the ONLY page that has the delegate, and create proposal button.

**Comment by adamgall** [comment]
> > Can probably delete the entire Proposals page too then, no?
> 
> @adamgall I was just about to do this, then saw it's literally the ONLY page that has the delegate, and create proposal button.

Ok yeah. So then please leave some comments about this on the Notion page.

**Comment by adamgall** [comment]
@DarksightKellar these items are already covered in https://github.com/decentdao/decent-interface/issues/2523 it looks like

**Comment by adamgall** [comment]
We can't merge this until the Delegate and New Proposal buttons, and the Proposals filter components, have been moved to the Home page.

**Review by mudrila — APPROVED** [review]
Lookz good - but yeah, need to get delegate / create proposal buttons on da dashboard first

**Comment by DarksightKellar** [comment]
Yeah I suppose we need some design input here on placement @decentdao/design ?

**Comment by nicolaus-sherrill** [comment]
@DarksightKellar yeah, that's it! I commented with this in another issue, too

**Review by Da-Colon — APPROVED** [review]

**Comment by parkermccurley** [comment]
Approved

**Pull request merged**